hack/install-agent-prerequisites.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# source: https://github.com/antrea-io/antrea/blob/main/docs/external-node.md#prerequisites-on-kubernetes-cluster
+
+set -e
+
+export CLUSTER_NAME=$(kubectl config current-context)
+
+case $CLUSTER_NAME in
+    *"eks"*)
+    export ANTREA_API_SERVER="https://$(kubectl get svc antrea -n kube-system -o jsonpath='{.status.loadBalancer.ingress[].hostname}')"
+    ;;
+    *"aks"*)
+    export ANTREA_API_SERVER=$(kubectl get svc antrea -n kube-system -o jsonpath='{.status.loadBalancer.ingress[].ip}')
+    ;;
+    *)
+    echoerr "Unknown cluster type. Only EKS and AKS are supported"
+    exit 1
+    ;;
+esac
+
+kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/v1.8.0/build/yamls/externalnode/vm-agent-rbac.yml
```

Comment:
This script needs to take in version as an input and download the rbac file from the respective branch/version.